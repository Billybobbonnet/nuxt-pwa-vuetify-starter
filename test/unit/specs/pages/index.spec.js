import { createLocalVue, shallowMount } from "@vue/test-utils";
import Vuetify from "vuetify";

import Index from "~/pages/index";

describe("Index", () => {
  let wrapper;

  beforeAll(() => {
    const localVue = createLocalVue();
    localVue.use(Vuetify);

    wrapper = shallowMount(Index, {
      localVue,
      stubs: ["nuxt-logo", "vuetify-logo"]
    });
  });

  it("renders to a snapshot", () => {
    expect(wrapper.element).toMatchSnapshot();
  });
});
