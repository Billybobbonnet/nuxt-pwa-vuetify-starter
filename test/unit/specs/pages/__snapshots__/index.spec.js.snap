// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index renders to a snapshot 1`] = `
<vlayout-stub
  align-center="align-center"
  column="column"
  justify-center="justify-center"
  tag="div"
>
  <vflex-stub
    md6="md6"
    sm8="sm8"
    tag="div"
    xs12="xs12"
  >
    <div
      class="text-xs-center my-5"
    >
      <nuxt-logo-stub />
       
      <vuetify-logo-stub />
    </div>
     
    <vcard-stub
      tag="div"
    >
      <vcardtitle-stub
        class="headline"
      />
       
      <vcardtext-stub>
        <p>
          Vuetify is a progressive Material Design component framework for Vue.js. It was designed to empower developers to create amazing applications.
        </p>
        <p>
          For more information on Vuetify, check out the 
          <a
            href="https://vuetifyjs.com"
            target="_blank"
          >
            documentation
          </a>
          .
        </p>
        <p>
          If you have questions, please join the official 
          <a
            href="https://chat.vuetifyjs.com/"
            target="_blank"
            title="chat"
          >
            discord
          </a>
          .
        </p>
        <p>
          Find a bug? Report it on the github 
          <a
            href="https://github.com/vuetifyjs/vuetify/issues"
            target="_blank"
            title="contribute"
          >
            issue board
          </a>
          .
        </p>
        <p>
          Thank you for developing with Vuetify and I look forward to bringing more exciting features in the future.
        </p>
         
        <div
          class="text-xs-right"
        >
          <em>
            <small>
              â€” John Leider
            </small>
          </em>
        </div>
         
        <hr
          class="mt-3"
        />
      </vcardtext-stub>
       
      <vcardactions-stub>
        <vspacer-stub />
         
        <vbtn-stub
          activeclass="v-btn--active"
          color="primary"
          flat="true"
          nuxt="nuxt"
          tag="button"
          to="/inspire"
          type="button"
        >
          Continue
        </vbtn-stub>
      </vcardactions-stub>
    </vcard-stub>
  </vflex-stub>
</vlayout-stub>
`;
